examples/dropwizard-app/build.gradle.kts

```diff
@@ -31,6 +30,7 @@ val execTask by tasks.register("start", JavaExec::class) {
     dependsOn("jar")
     classpath = sourceSets.main.get().runtimeClasspath
     jvmArgs = listOf(
+            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005",
```

Comment:
I don't know why remote debugging arg is necessary here

Reply:
It is not, I removed it in c31a8d7 . Good catch!